tasks/src/main/scala/dagr/tasks/misc/VerifyBamId.scala

```diff
@@ -0,0 +1,31 @@
+package dagr.tasks.misc
+
+import java.nio.file.Path
+
+import dagr.core.config.Configuration
+import dagr.core.execsystem.{Cores, Memory}
+import dagr.core.tasksystem.{FixedResources, ProcessTask}
+import dagr.tasks.{FilenamePrefix, PathToBam, PathToVcf}
+
+import scala.collection.mutable.ListBuffer
+
+/**
+  * Class for running verifyBamID on sequence data in genotype-free mode.
+  */
+class VerifyBamId(val vcf: PathToVcf, val bam: PathToBam, val out: Path,
```

Comment:
can you put each constructor parameter on its own line so that it is easier to read their defaults?